jest.config.js

```diff
@@ -26,5 +26,6 @@ module.exports = {
   ],
   transform: {
     '^.+\\.jsx?$': path.join(__dirname, '/lib/babel-jest'),
+    '.+\\.(css|styl|less|sass|scss)$': 'jest-transform-css',
```

Comment:
Jest requires manual configuration for CSS transpilation. I found this perusing some Jest issue, but I'm not quite sure if this is _our_ preferred way of doing this?